pkg/kab/patch_manifest.go

```diff
@@ -70,11 +70,17 @@ func (c *Client) applyLabels(res *v1alpha1.KabResource) (content string, e error
 
 	log.Tracef("Applying labels resource: %s Labels: %+v...", res.Name, res.Labels)
 
-	path, err = url.Parse(res.Path)
-	if err != nil {
-		return "", err
+	var byteContent []byte
```

Comment:
Rather than each patch needing to support both `.Content` and `.Path`, how about we resolve paths once when loading the manifest and then only operate on `.Content` for all further interaction?

Reply:
done!